tests/DataFactories/WooCommerceProduct.php

```diff
@@ -50,6 +52,13 @@ function withSku($sku) {
     return $this->update('sku', $sku);
   }
 
+  /**
+   * @return $this
+   */
+  function withRandomSku() {
```

Comment:
I think we can get rid of this method. In constructor we can set SKU to null and then in `create` method we can test if it is still `null` and in case it is we just generate random one there.